elementary/monitor/cli.py

```diff
@@ -291,13 +292,16 @@ def send_report(
     anonymous_tracking.track_cli_start('monitor-send-report', get_cli_properties(), ctx.command.name)
     try:
         config.validate_send_report()
+        # If file name contains a folders (is a path),
+        # we want to make sure that we create the report localy without the folders of the full path.
+        local_file_path = os.path.basename(file_name) if file_name else None
```

Comment:
Maybe we should add a new param called bucket_folder? then if you want to upload to bucket you can mention - 
bucket_name, bucket_folder and a file_name - wdyt?

Reply:
I am not sure I am up to that idea..
Both S3 and GCS works in the way that you provide the path as part of the file name.
If you work with buckets, you should be familiar with that and I don't want to create a different behaviour.
Plus it can start getting weird that if you have folders hierarchy you provide a path under the bucket_folder and not just a folder name. In that case, what is the difference from passing it to the file name?